lib/galaxy/util/__init__.py

```diff
@@ -335,6 +335,17 @@ def shrink_string_by_size( value, size, join_by="..", left_larger=True, beginnin
     return value
 
 
+def parse_query_string( query_string ):
```

Comment:
I don't think it will be necessary, but there's absolutely no harm in preventing malice.